environments/kolla/files/overlays/horizon/custom_local_settings

```diff
@@ -1,4 +1,11 @@
-# customer specific configuration
+WEBSSO_ENABLED = True
 
-# SESSION_TIMEOUT = 7200
-# OPENSTACK_ENABLE_PASSWORD_RETRIEVE = True
+WEBSSO_KEYSTONE_URL = "http://192.168.16.10:5000/v3"
```

Comment:
This should probably be the external API address of keystone: 192.168.96.9.

Reply:
Well then I will change the address here and  
in `environments/kolla/files/overlays/keystone/wsgi-keystone.conf`  for ` OIDCRedirectURI`,  
and in `environments/custom/playbook-keycloak-oidc-client-config.yml` for `keystone_redirect_uris`.

Reply:
Done